server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/RemoteDeliveryConfiguration.java

```diff
@@ -213,9 +213,6 @@ public Properties createFinalJavaxProperties() {
         // Reactivated: javamail 1.3.2 should no more have problems with "250 OK" messages
         // (WAS "false": Prevents problems encountered with 250 OK Messages)
         props.put("mail.smtp.ehlo", "true");
-        // By setting this property to true the transport is allowed to send 8 bit data to the server (if it supports
-        // the 8bitmime extension).
-        props.setProperty("mail.smtp.allow8bitmime", "true");
```

Comment:
Shouldn't we document this behavioral change somewhere?

Reply:
`upgrade-instructions.md` is it okay?

Reply:
Maybe more changelog entry mentionning the JIRA, there is nothing to be done upon upgrade.

Reply:
I added and squashed